Exercises/Google-Page/Index.html

```diff
@@ -3,8 +3,51 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Google Search</title>
+        <link rel="stylesheet" href="Style.css"/>
 </head>
 <body>
-    
+                <nav>
+                    <div>
+                    <a href="https://about.google">About</a>
+                    <a href="https://store.google.com/">Store</a>
+                    </div>
+                    <div class="nav-right">
+                    <a href="https://accounts.google.com/">Gmail</a>
+                    <a href="https://images.google.com/">Images</a>
+                    <img src="menu.png" height="15px" width="15px"/>
+                        <a href="https://accounts.google.com/"> 
+                                <img src="Nikhil Profile Pic.jpg" class="profile-pic">
+                        </a>
+                    </div>
+                </nav>
+                <main>
+                        <img class="google_logo" src="armenia-independence-day-2020-6753651837108541-2xa.gif" alt="Google logo"/>
+
+                        <form action="https://google.ca/search/" method="GET">
+                                <div class="search-container">
+                                        <img src="search-icon.png" alt="Search Icon"/>
+                                        <input type="text" name="q"/>
+                                        <img src="voice-icon.png" alt="Voice Icon"/>
+                                </div>
+                                <div class="button-container">
+                                        <button>Google Search</button>
+                                        <button>I'm Feeling Lucky</button>
+                                </div>
+                        </form>                 
+                </main>
+                <footer>
+                        <div>
+                                <a href="#">Canada</a>
+                                <a href="#">Advertising</a>
+                                <a href="#">Business</a>
+                                <a href="#">How Search works</a>
+                        </div>
+                        <div class="footer-right">
+                                <a href="#">Privacy</a>
+                                <a href="#">Terms</a>
+                                <a href="#">Settings</a>
+                        </div>
+                </footer>
```

Comment:
Footer doesn't look as it was shown in example img.
https://github.com/chriztie/react-comit/blob/master/Prerequisites/CSS/09-css-google/result-final.png
Check and think how it can be done.